test/byContract/IexecPocoBoost/IexecPocoBoostDelegate.test.ts

```diff
@@ -1,234 +1,168 @@
+    let [appAddress, datasetAddress, workerpoolAddress]: string[] = [];
+    let [
         admin,
         requester,
         sponsor,
         beneficiary,
-        appProvider,
-        datasetProvider,
-        scheduler,
-        worker: worker1,
-        worker1,
-        worker2,
-        enclave,
-        anyone,
-        teeBroker,
-    };
-}
-
-describe('IexecPocoBoostDelegate', function () {
-    let iexecPocoBoostInstance: MockContract<IexecPocoBoostDelegate>;
-    let oracleConsumerInstance: FakeContract<TestClient>;
-    let gasWasterClientInstance: MockContract<GasWasterClient>;
-    let appInstance: MockContract<App>;
-    let workerpoolInstance: MockContract<Workerpool>;
-    let datasetInstance: MockContract<Dataset>;
-    let appRegistry: FakeContract<AppRegistry>;
-    let datasetRegistry: FakeContract<DatasetRegistry>;
-    let workerpoolRegistry: FakeContract<WorkerpoolRegistry>;
-    let someContractInstance: MockContract<TestClient>;
-    let [
         appProvider,
         datasetProvider,
         scheduler,
         worker,
         enclave,
-        requester,
-        sponsor,
-        beneficiary,
         anyone,
         teeBroker,
-    ] = [] as SignerWithAddress[];
+    ]: SignerWithAddress[] = [];
     let ordersActors: OrdersActors;
     let ordersAssets: OrdersAssets;
     let ordersPrices: OrdersPrices;
+    let domain: TypedDataDomain;
 
-    beforeEach('set up contract instances and mock app', async () => {
-        const fixtures = await loadFixture(deployBoostFixture);
-        iexecPocoBoostInstance = fixtures.iexecPocoBoostInstance;
-        appProvider = fixtures.appProvider;
-        datasetProvider = fixtures.datasetProvider;
-        scheduler = fixtures.scheduler;
-        worker = fixtures.worker;
-        enclave = fixtures.enclave;
-        requester = fixtures.requester;
-        sponsor = fixtures.sponsor;
-        beneficiary = fixtures.beneficiary;
-        anyone = fixtures.anyone;
-        teeBroker = fixtures.teeBroker;
+    beforeEach('Deploy', async () => {
+        proxyAddress = await loadHardhatFixtureDeployment();
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        const accounts = await getIexecAccounts();
+        ({
+            iexecAdmin: admin,
+            requester,
+            sponsor,
+            beneficiary,
+            appProvider,
+            datasetProvider,
+            scheduler,
+            worker,
+            enclave,
+            anyone,
+            sms: teeBroker,
+        } = accounts);
+        iexecWrapper = new IexecWrapper(proxyAddress, accounts);
+        domain = iexecWrapper.getDomain();
+        ({ appAddress, datasetAddress, workerpoolAddress } = await iexecWrapper.createAssets());
+        iexecPocoBoostInstance = IexecPocoBoostDelegate__factory.connect(proxyAddress, anyone);
+        iexecMaintenanceAsAdmin = IexecMaintenance__factory.connect(proxyAddress, admin);
```

Comment:
Why using admin instead of directly iexecAdmin

Reply:
It was to avoid diff, but we can simplify things thanks :) : [Remove global admin variable](https://github.com/iExecBlockchainComputing/PoCo/pull/148/commits/9928a0206743e166e6931a0d5451e02cac933faa)